src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -37,50 +39,24 @@ storiesOf('JsonSchemaViewer', module)
     <JsonSchemaViewer
       schema={object('schema', {})}
       defaultExpandedDepth={number('defaultExpandedDepth', 0)}
-      onGoToRef={action('onGoToRef')}
       maxRows={number('maxRows', 5)}
       mergeAllOf={boolean('mergeAllOf', true)}
     />
   ))
-  .add('custom row renderer', () => {
-    const customRowRenderer: RowRenderer = (node, rowOptions) => {
-      return (
-        <>
-          <SchemaRow treeListNode={node} rowOptions={rowOptions} />
-          <Flex h="full" alignItems="center">
-            <Button pl={1} mr={1} size="sm" appearance="minimal" icon="issue" />
-            <input type="checkbox" />
-          </Flex>
-        </>
-      );
-    };
-
-    return (
-      <JsonSchemaViewer
-        schema={object('schema', schema as JSONSchema4)}
-        onGoToRef={action('onGoToRef')}
-        maxRows={number('maxRows', 5)}
-        mergeAllOf={boolean('mergeAllOf', true)}
-        rowRenderer={customRowRenderer}
-      />
-    );
-  })
   .add('stress-test schema', () => (
     <>
-      <div style={{ height: 345 }}>
+      <div style={{ height: 345, overflowY: 'scroll' }}>
         <JsonSchemaViewer
           schema={stressSchema as JSONSchema4}
           defaultExpandedDepth={number('defaultExpandedDepth', 2)}
-          onGoToRef={action('onGoToRef')}
```

Comment:
Don't we need that anymore?
It was built for Studio so that you could activate a local source node